Jenkinsfile

```diff
@@ -55,14 +55,27 @@ node {
 onlyOnMaster {
     milestone()
     stage('qa deploy') {
-        deployApp(image: img, app: 'h', env: 'qa')
+        deployApp(image: img, app: 'h', env: 'qa', region: "us-west-1")
     }
 
     milestone()
-    stage('prod deploy') {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: 'h', env: 'prod')
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
```

Comment:
Introduced dedicated `approval` stage. This has been done for Via already, and I think the change is beneficial. It makes what is going on, and user input requirements more obvious:

<img width="798" alt="Screenshot 2021-06-28 at 11 00 52" src="https://user-images.githubusercontent.com/18003117/123618788-6b7ebe00-d800-11eb-8da5-792f16007e98.png">

Reply:
👍